packages/block-library/src/rss/index.php

```diff
@@ -66,7 +66,7 @@ function render_block_core_rss( $attributes ) {
 		$excerpt = '';
 		if ( $attributes['displayExcerpt'] ) {
 			$excerpt = html_entity_decode( $item->get_description(), ENT_QUOTES, get_option( 'blog_charset' ) );
-			$excerpt = esc_attr( wp_trim_words( $excerpt, $attributes['excerptLength'], ' [&hellip;]' ) );
```

Comment:
I noticed this is being escaped early with `esc_attr` and then again later with `esc_html`. I assume the `esc_attr` is an error but would appreciate a confidence check @hellofromtonya

Reply:
Or anyone really 😄

Reply:
Looks good—no need to double escape here. Btw `esc_html` and `esc_attr` are the same under the hood.

Reply:
Really!? Why then do we have both? 🤔

Reply:
Probably for backward compatibility, plus it's easier to remember escape function names this way.